packages/graph/hash_graph/lib/graph/src/store/postgres/query/compile.rs

```diff
@@ -69,13 +69,22 @@ impl<'f: 'q, 'q, T: PostgresQueryRecord<'q>> SelectCompiler<'f, 'q, T> {
 
     /// Adds a new path to the selection.
     ///
-    /// Optionally, a path can be ordered by passing an [`Ordering`] alongside the path.
-    pub fn add_selection_path(&mut self, path: &'q T::Path<'q>, ordering: Option<Ordering>) {
+    /// Optionally, a path can be ordered by passing an [`Ordering`] alongside the path. Also, when
+    /// `distinct` is `true`, this path will be selected distinctly.
```

Comment:
Nit: I wonder if this is misleasing, as it would only be selected distinctly if it's the sole field. Otherwise it would be part of a distinctness "pattern". I get what is meant by this, but perhaps we should make it explicit :)

Reply:
With the new change below I changed the documentation as well: c06712685401198a60e61298a87d4acfd05ee604